
# Position Controllers ---------------------------------------
joint1_position_controller:
  command_topic: joint1_velocity_controller/command
  joint: link0_to_link1
  pid: {p: 15.0, i: 0.0, d: 0.0, high_limit: 6, low_limit: -6}
joint2_position_controller:
  command_topic: joint2_velocity_controller/command
  joint: link1_to_link2
  pid: {p: 45.0, i: 15.0, d: 0.0, high_limit: 6, low_limit: -6}
joint3_position_controller:
  command_topic: joint3_velocity_controller/command
  joint: link2_to_link3
  pid: {p: 25.0, i: 10.0, d: 0.0, high_limit: 6, low_limit: -6}
joint4_position_controller:
  command_topic: joint4_velocity_controller/command
  joint: link3_to_link4
  pid: {p: 20.0, i: 2.0, d: 0.0, high_limit: 6, low_limit: -6}
joint5_position_controller:
  command_topic: joint5_velocity_controller/command
  joint: link4_to_link5
  pid: {p: 5.0, i: 1.0, d: 0.0, high_limit: 6, low_limit: -6}
  #pid: {p: 10.0, i: 1.0, d: 0.0, high_limit: 6, low_limit: -6}
